src/management/__generated/managers/ActionsManager.ts

```diff
@@ -0,0 +1,810 @@
+/* tslint:disable */
```

Comment:
Would be nice to run prettier / and ts/eslint fix over these files when we generate them